history-of-the-quakers-v3/original/25-book-6-chapter-5.adoc

```diff
@@ -212,9 +215,6 @@ to impose upon them, and tempt them to evil, are not principally accountable,
 for the crimes they commit, and their consequences,
 is a query worthy of their most serious consideration.
 {footnote-paragraph-split}
-From Maija:
-Here`'s the footnote within a footnote
-{footnote-paragraph-split}
 *At the treaty at Carlisle,
```

Comment:
Here's a footnote within a footnote that I marked off with an asterisk.

Reply:
Ok, this actually turned out to look fairly decent in the PDF, so I think we can leave it.  However, lets escape it, so it's `+++*+++At the treaty at Carlisle,` instead 👍